app/models/classification.rb

```diff
@@ -465,7 +471,9 @@ def validate_uniqueness_on_tag_name
   end
 
   def validate_format_of_name
-    errors.add("name", "must be lowercase alphanumeric characters and underscores without spaces") unless (name =~ /[^a-z0-9_:]/).nil?
+    unless (name =~ /[^a-z0-9_:]/).nil?
+      errors.add("name", "must be lowercase alphanumeric characters, colons and underscores without spaces")
+    end
```

Comment:
This is 2 month old but I just realized why it's not a standard validator in the first place — `name` is not a column of classification, it's computed by chunking the associated `tag`'s name.
Don't ask me why the Classification model is doing Tag validations, or why they're separate tables at all :-)